gymnasium/vector/vector_env.py

```diff
@@ -127,7 +127,7 @@ class VectorEnv(Generic[ObsType, ActType, ArrayType]):
     def reset(
         self,
         *,
-        seed: int | None = None,
+        seed: int | list[int] | None = None,
```

Comment:
This was an intentional decision that is overwritten by the `AsyncVectorEnv` and `SyncVectorEnv` 
but not for `VectorEnv`

Reply:
Ah gotcha, will revert.

Reply:
Thanks